Update screenshot errors inbox (#18766)
newrelic/docs-website: src/content/docs/errors-inbox/error-users-impacted.mdx

freshnessValidatedDate: never
---

> New Relic agents automatically capture data about user impact.When you organize by error group, you can see how many users are affected by errors. This helps you prioritize where to focus your debugging efforts by showing you the errors with the highest numbers of users affected.

Author: ```suggestion
New Relic agents automatically capture data about user impact. When you organize by error group, you can see how many users are affected by errors. This helps you prioritize where to focus your debugging efforts by showing you the errors with the highest numbers of users affected.
```

<img
    title="An image showing how to navigate users impacted"
    alt="This is an image of the main errors inbox users impacted"
    src="/images/errors-inbox_screenshot-full_users-impacted.webp"
/>
<figcaption>
    Go to **[one.newrelic.com](https://one.newrelic.com) > APM & Services > Errors (errors inbox) > Triage**: You can find users impacted in the second column of the **Error groups** table, located beneath the **Error count** and **Error rate** time series.   
</figcaption>

## Custom attributes to track users impacted  [#custom-attributes]

> In some cases, an error event may not have attributes that identify the end user affected by that error event. You can manually add custom attributes so our agents captures user-identifying metadata at the time of ingest. 

Reviewer: You can manually add custom attributes so our agents capture .....

> In some cases, an error event may not have attributes that identify the end user affected by that error event. You can manually add custom attributes so our agents captures user-identifying metadata at the time of ingest. 

Author: ```suggestion
In some cases, an error event may not have attributes that identify the end user affected by that error event. You can manually add custom attributes so our agents capture user-identifying metadata at the time of ingest. 
```

For cases where you need to add a custom attribute, we recommend reviewing [OpenTelemetry's attribute conventions](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction) because there are multiple common conventions for naming end user attributes.

> For example, one [OpenTelemetry standard](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/#general-identity-attributes) uses `enduser.id`. We'll use the following attributes in order of priority:

Author: ```suggestion
For example, [OpenTelemetry standards](https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/#general-identity-attributes) uses the following attributes in order of priority:
```

    * `userId`
    * `user`

> You can review how New Relic processes custom attributes by reviewing our [Collect custom events doc](/docs/data-apis/custom-data/custom-events/collect-custom-attributes/#enabling-custom).

Reviewer: ...reviewing our [collect custom events doc] ......
